server/routerlicious/packages/local-server/package.json

```diff
@@ -87,6 +87,7 @@
 		"concurrently": "^8.2.1",
 		"copyfiles": "^2.4.1",
 		"eslint": "~8.55.0",
+		"events_pkg": "npm:events@^3.1.0",
```

Comment:
This shouldn't be a dev dependency since it's used in src/localWebSocketServer.ts which is part of the "prod code" for the package. It should completely replace the "events" dep above. Same for the other packages.